samples/TagHelperSample.Web/Areas/Admin/Views/Admin/Index.cshtml

```diff
@@ -0,0 +1,7 @@
+﻿@*
+    For more information on enabling MVC for empty projects, visit http://go.microsoft.com/fwlink/?LinkID=397860
+*@
+@{
+}
+
+<a asp-action="Index" asp-controller="Home" asp-area="">Go home!</a>
```

Comment:
Thinking outside the box, there could be a different property to explicitly indicate "no area", e.g. `bool asp-use-main-area { get; set; }`. If that is `true` then we add `area = ""` to the route values. If it's `false` (or unset), use the newly added behavior here. If it's `true` and `asp-area` or asp-route-area` is set then we could consider throwing (or I'm a fan of silently ignoring because who cares).

Thoughts on this?